packages/zoe/src/zoeService/makeInvitation.js

```diff
@@ -20,31 +29,41 @@ export const createInvitationKit = shutdownZoeVat => {
     assert.typeof(installation, 'object');
 
     /** @type {ZoeInstanceAdminMakeInvitation} */
-    const makeInvitation = (
+    const makeInvitation = async (
       invitationHandle,
       description,
       customProperties,
+      relativeFee = undefined,
+      relativeExpiry = undefined,
     ) => {
       assert.typeof(invitationHandle, 'object');
       assert.typeof(
         description,
         'string',
         X`The description ${description} must be a string`,
       );
+      const absoluteFee = translateFee(relativeFee);
+      const absoluteExpiry = await translateExpiry(relativeExpiry);
+
+      let feeInfo = {};
+      if (absoluteFee && absoluteExpiry) {
+        feeInfo = { fee: absoluteFee, expiry: absoluteExpiry };
+      }
```

Comment:
(This one is really important so making visible prior to review.) 

As of this PR, Fred will take the `fee` and `expiry` properties in the invite amount to be authentic. At least to have been translated by Zoe's internal `translateFee` and `translateExpiry` and have whatever invariants follow from that. However, by allowing the `feeInfo` to be empty, it would fail to override bad `fee` and `expiry` fields provided by the contract in the description.

```suggestion
      const feeInfo = { fee: absoluteFee, expiry: absoluteExpiry };
```

Reply:
good catch!

Reply:
Fixed